### ISSUE TITLE
A corrupt text on 'History'

### ISSUE THREAD

**Issue by meoldae** [Swati4star/Images-to-PDF#1033]
I checked that corrupted text in 'History'.
Like this picture

![Uploading Issue.png…]()

Can i work on this issue?

**Comment by meoldae** [comment]
@codegsaini 
I want to solve it by increasing the compartment or reducing the font size.
Can I work on this Issue?

**Comment by codegsaini** [comment]
@meoldae Yes, please! It will be great and your contribution will be appreciated :)

Suggestion - Don't ask anyone not even the author to solve any issue already present in the issue list or if you found any new issue. Just follow these steps - 

### Case 1 - If you found any new issue
- If you found any new issue just create/submit that issue and if you want to work on it then solve it and create pull request.
- To be in safe side so that other not work on the same, you should write just a line in the description that **_I am working on it_.**
- By doing so, author will (if he/she have to) assign you for that issue and not anyone else.
**( I must clear that, as per my concern, anyone can propose solution of any issue regardless of to whom it is assigned. )**

### Case 2 - If you want to work on already mentioned issue
- If you want to solve issue already present in the issue list. Then follow these steps - 
  - Check if it is already assigned to anyone else. (Even if it is assigned to anyone else, you can work on it but in future you both will solve the same issue and it will be a waste of time for any one solver)
  - After you found that no one is assigned to that issue, immediately start solving it (if you want to do so).
  - To be in safe side just comment below that issue that **_I am working on it._**
  
## In that manner you don't have to ask anyone if you can work on that issue or not.
(Disclaimer- This is my personal opinion. These are neither stated in the official guideline of this project nor of Github.)

**Comment by Swati4star** [comment]
Agree with @codegsaini.
